nerfstudio/models/mipnerf.py

```diff
@@ -151,6 +154,8 @@ def get_image_metrics_and_images(
         rgb_fine = outputs["rgb_fine"]
         acc_coarse = colormaps.apply_colormap(outputs["accumulation_coarse"])
         acc_fine = colormaps.apply_colormap(outputs["accumulation_fine"])
+
+        assert self.config.collider_params is not None
```

Comment:
- With the way the CLI is set up right now the users actually can't unset collider params for mipnerf (although the type annotations do allow for it ☹️)
- We could add another assert to `__init__`, but we would still need this one for the type checker to pass, is that what you meant?

Reply:
1) I understand.
2) Yes, that is what I meant. I think it can be a good practice to declare required contracts in __init__ such that it is clear for someone who reads the code/wants to modify it.